build/nuget/build.ps1

```diff
@@ -21,7 +21,7 @@ if(!$Repo) {
 
 $version = "0.0.1"
```

Comment:
Like Brian mentioned.  this is prototyping.  
We are testing out the workflow using PM's as guinea pigs.

---
In reply to: [88757560](https://github.com/Microsoft/WinObjC/pull/1426#discussion_r88757560) [](ancestors = 88757560)